x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1-r1.ebuild

```diff
@@ -10,13 +10,16 @@ SRC_URI="https://github.com/majn/telegram-purple/releases/download/v${PV}/telegr
 LICENSE="GPL-2+"
 KEYWORDS="~amd64 ~x86"
 SLOT="0"
-IUSE="gcrypt +nls +webp"
+IUSE="gcrypt libressl +nls +webp"
```

Comment:
second time I tell you to revbump the ebuild once you change the IUSE, if you cannot follow what you've been told by at least 2 different people, please do not contribute

Reply:
So here's third voice that says you don't need to revbump when adding USE flags to ~arch package.

Reply:
@mgorny how did you count? ;) Amy and I already told him to change the revision for scheme things

Reply:
Hi @Zlogene, thanks for the comment. I try to follow what's written in devmanual, and existing Gentoo practices. For sure sometimes I'm mistaken, I'm still new to Gentoo. You and Amy Liffey asked to revbump the ebuild for dev-scheme/gambit (https://github.com/gentoo/gentoo/pull/9929), and I did as you asked. And I understand the logic: always revbump stable ebuilds, even with a slightest change (although it's not _obvious_ from the devmanual [for me]).

Yet I sent about 5 patches that add USE=libressl to unstable packages without changing revisions, approved by like 2 or 3 different developers. There are much more in the tree from other contributors, so I think it's a common practice. And from the devmanual it is not obvious [for me] why to revbump every time I add a USE flag that makes no difference to systems with the _unstable_ package already installed.

If you still insist that the revision should be changed in this case, let's suppose you are right. But maybe it's a good idea to do something with this bug then? https://bugs.gentoo.org/223153

PS. Excuse me, but I'm not going to stop contributing. I like it.